frontend/build-configs/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "build-configs",
+  "version": "0.0.1",
+  "private": true,
+  "main": "dist/index.js",
+  "scripts": {
+    "preinstall": "tsc --build"
+  }
```

Comment:
```suggestion
  "main": "index.ts",
```

Reply:
this does not work for me @sarahsporck 
When i do an `npm i && npm run build` in `administration`, i'm facing the following error

```js
/Users/afischer/projects/entitlementcard/administration/node_modules/build-configs/index.ts:1
export {default as buildConfigNuernberg} from './nuernberg'
^^^^^^

SyntaxError: Unexpected token 'export'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1149:20)
    at Module._compile (node:internal/modules/cjs/loader:1190:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1280:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/afischer/projects/entitlementcard/administration/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1089:32)
    at Function.Module._load (node:internal/modules/cjs/loader:930:12)
    at Module.require (node:internal/modules/cjs/loader:1113:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/Users/afischer/projects/entitlementcard/administration/src/project-configs/bayern/deepLinkConfig.ts:1:1)

```

Reply:
I'll review again tomorrow. But you might be right. Still I'm wondering why it is working for integreate. Well, we'll see tomorrow :)

Reply:
yes i also checked the integreat implementation and i tried for some time to figure out why this is working there and not here

Reply:
It seems the problem lies in cra not supporting [project references](https://github.com/facebook/create-react-app/issues/10695), and thus our build script (which originally was a cra script) does not support project references https://github.com/facebook/create-react-app/issues/10695. An easy workaround for now is to run tsc --build before run. I'll create an PR for you to visualize the changes. The current solution requires typescript to be installed globally and is therefore not sufficent in my opinion. However I think we should consider moving to a simpler webpack setup (as @michael-markl already [suggested in an older pr](https://github.com/digitalfabrik/entitlementcard/pull/1026#issue-1773363300)) at some point. Moving to ts-loader could also resolve this issue.